examples/documentation.html

```diff
@@ -327,7 +327,7 @@
 
         <div style="height: 50px"></div>
 
-		<h3 class="option"><span class="expand all">[ <a href="javascript:$('#optionlist>ul').toggle()">toggle all</a> ]</span></h3>
+    <h3 class="option"><span class="expand all">[ <a href="javascript:$('#optionlist>ul').toggle()">toggle all</a> ]</span></h3>

```

Comment:
The commit is not about indentation, it rather adds a useful api information to the documentation. But even if we take a closer look to that indentation at https://github.com/novus/nvd3/blob/master/examples/documentation.html#L330. It obviously looks incorrect, as it's not nested and being placed between two closed `div`s.